src/Classes/ServiceAPI/MyRadio_User.php

```diff
@@ -2216,6 +2365,10 @@ public static function getBulkAddForm()
      */
     public function toDataSource($mixins = [])
     {
+        if ($this->getHideProfile()){
+            return $this->getEmptyData();
+        }
+
```

Comment:
ok, yeah i guess this is a way of doing it. maybe we should discuss whether this way, or just a parameter saying 'profileHidden': bool, so toDataSource could still be used?